### PR TITLE
cleanup code and update dependencies

### PR DESCRIPTION
**Pull request by LarsBuur**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
+		"build-stats": "cross-env ANALYZE=true npm run build",
+		"export": "next export",
+		"build-prod": "run-s clean build export",
+		"clean": "rimraf .next out",
+		"lint": "next lint",
+		"lint-fix": "next lint --fix",
+		"build-types": "tsc --noEmit --pretty"
+	},
+	"dependencies": {
+		"classnames": "^2.3.1",
```
Comment (i-am-mani) [inline package.json:17]: we use `clsx`

**Review by Revilo321 — APPROVED** [review]
LGTM

File: package.json [hunk 2]
```diff
@@ -1,55 +1,52 @@
...
+		"next": "^12.0.9",
+		"next-seo": "^4.29.0",
+		"react": "^17.0.2",
+		"react-dom": "^17.0.2",
+		"styled-jsx-plugin-postcss": "^4.0.1"
```
Comment (i-am-mani) [inline package.json:22]: We don't use this, i suppose we can remove this as well?

File: package.json [hunk 3]
```diff
@@ -1,55 +1,52 @@
...
-    "npm-run-all": "^4.1.5",
-    "postcss": "^8.4.5",
-    "prettier": "^2.5.1",
-    "rimraf": "^3.0.2",
-    "tailwindcss": "^3.0.17",
-    "typescript": "^4.5.5"
-  },
-  "license": "ISC"
-}
+	"name": "next-js-boilerplate",
```
Comment (i-am-mani) [inline package.json:2]: should we pick a better name?

File: package.json (hunk 1 above)
Comment (LarsBuur) [reply package.json]: removed.

File: package.json (hunk 2 above)
Comment (LarsBuur) [reply package.json]: removed

File: package.json (hunk 3 above)
Comment (LarsBuur) [reply package.json]: agree

**Pull request merged**